conf/config.yaml

```diff
@@ -67,6 +67,7 @@ nginx_config:                     # config for render the template to genarate n
     client_header_timeout: 60s     # timeout for reading client request header, then 408 (Request Time-out) error is returned to the client
     client_body_timeout: 60s       # timeout for reading client request body, then 408 (Request Time-out) error is returned to the client
     send_timeout: 10s              # timeout for transmitting a response to the client.then the connection is closed
+    underscores_in_headers: "on"   # default enables the use of underscores in client request header fields
```

Comment:
We should update bin/apisix` source file.

Reply:
I forgot the test `bin/apisix init`, so I didn't notice that...